From 8b3d0ccc2381919c8de944caa6485eb88d13e819 Mon Sep 17 00:00:00 2001
From: Thomas Robitaille <thomas.robitaille@gmail.com>
Date: Thu, 6 Jun 2019 09:49:48 +0200
Subject: [PATCH] Fix import from collections

---
 astropy/utils/timer.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/astropy/utils/timer.py b/astropy/utils/timer.py
index 30519b9f681..c9d0bacdb5f 100644
--- a/astropy/utils/timer.py
+++ b/astropy/utils/timer.py
@@ -4,7 +4,8 @@
 # STDLIB
 import time
 import warnings
-from collections import Iterable, OrderedDict
+from collections import OrderedDict
+from collections.abc import Iterable
 from functools import partial, wraps
 
 # THIRD-PARTY
