diff -Naur astropy-5.0/astropy/config/configuration.py astropy-5.0.new/astropy/config/configuration.py
--- astropy-5.0/astropy/config/configuration.py	2021-11-19 15:17:19.000000000 +0100
+++ astropy-5.0.new/astropy/config/configuration.py	2021-12-26 15:49:25.160408248 +0100
@@ -20,7 +20,7 @@
 from warnings import warn
 from contextlib import contextmanager, nullcontext
 
-from astropy.extern.configobj import configobj, validate
+import configobj, validate
 from astropy.utils import find_current_module, silence
 from astropy.utils.decorators import deprecated
 from astropy.utils.exceptions import AstropyDeprecationWarning, AstropyWarning
