diff -ur astropy-0.3.2/astropy/config/configuration.py astropy-0.3.2.six/astropy/config/configuration.py
--- astropy-0.3.2/astropy/config/configuration.py	2014-05-19 11:41:00.629963008 +0200
+++ astropy-0.3.2.six/astropy/config/configuration.py	2014-05-19 11:33:31.764331943 +0200
@@ -10,7 +10,7 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
+import six
 
 import inspect
 import pkgutil
@@ -23,7 +23,7 @@
 from os import path
 from warnings import warn
 
-from ..extern import six
+import six
 import configobj, validate
 from ..utils.exceptions import AstropyWarning
 
diff -ur astropy-0.3.2/astropy/config/paths.py astropy-0.3.2.six/astropy/config/paths.py
--- astropy-0.3.2/astropy/config/paths.py	2014-05-19 11:41:00.626962984 +0200
+++ astropy-0.3.2.six/astropy/config/paths.py	2014-05-19 11:33:31.661331111 +0200
@@ -59,7 +59,7 @@
             homedir = decodepath(os.path.join(os.environ['USERPROFILE']))
         else:
             try:
-                from ..extern.six.moves import winreg as wreg
+                from six.moves import winreg as wreg
                 shell_folders = r'Software\Microsoft\Windows\CurrentVersion\Explorer\Shell Folders'
                 key = wreg.OpenKey(wreg.HKEY_CURRENT_USER, shell_folders)
 
diff -ur astropy-0.3.2/astropy/constants/constant.py astropy-0.3.2.six/astropy/constants/constant.py
--- astropy-0.3.2/astropy/constants/constant.py	2014-05-19 11:41:00.637963073 +0200
+++ astropy-0.3.2.six/astropy/constants/constant.py	2014-05-19 11:33:32.134334929 +0200
@@ -1,7 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
+import six
 
 import functools
 import types
diff -ur astropy-0.3.2/astropy/constants/tests/test_constant.py astropy-0.3.2.six/astropy/constants/tests/test_constant.py
--- astropy-0.3.2/astropy/constants/tests/test_constant.py	2014-05-19 11:41:00.637963073 +0200
+++ astropy-0.3.2.six/astropy/constants/tests/test_constant.py	2014-05-19 11:33:32.223335648 +0200
@@ -1,7 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ...extern import six
+import six
 
 from .. import Constant
 from ...units import Quantity as Q
diff -ur astropy-0.3.2/astropy/coordinates/angles.py astropy-0.3.2.six/astropy/coordinates/angles.py
--- astropy-0.3.2/astropy/coordinates/angles.py	2014-05-19 11:41:00.510962044 +0200
+++ astropy-0.3.2.six/astropy/coordinates/angles.py	2014-05-19 11:33:25.225279159 +0200
@@ -12,7 +12,7 @@
 
 import numpy as np
 
-from ..extern import six
+import six
 from . import angle_utilities as util
 from .. import units as u
 from ..utils import deprecated, isiterable
diff -ur astropy-0.3.2/astropy/coordinates/builtin_systems.py astropy-0.3.2.six/astropy/coordinates/builtin_systems.py
--- astropy-0.3.2/astropy/coordinates/builtin_systems.py	2014-05-19 11:41:00.507962019 +0200
+++ astropy-0.3.2.six/astropy/coordinates/builtin_systems.py	2014-05-19 11:33:25.127278368 +0200
@@ -12,7 +12,7 @@
 
 import numpy as np
 
-from ..extern import six
+import six
 from .angles import Angle
 from .coordsystems import SphericalCoordinatesBase
 from ..time import Time
diff -ur astropy-0.3.2/astropy/coordinates/coordsystems.py astropy-0.3.2.six/astropy/coordinates/coordsystems.py
--- astropy-0.3.2/astropy/coordinates/coordsystems.py	2014-05-19 11:41:00.505962003 +0200
+++ astropy-0.3.2.six/astropy/coordinates/coordsystems.py	2014-05-19 11:33:25.086278037 +0200
@@ -8,7 +8,7 @@
 
 from abc import ABCMeta, abstractproperty, abstractmethod
 
-from ..extern import six
+import six
 from .. import units as u
 from .angles import Longitude, Latitude, Angle
 from .distances import Distance, CartesianPoints, cartesian_to_spherical, spherical_to_cartesian
diff -ur astropy-0.3.2/astropy/coordinates/matching.py astropy-0.3.2.six/astropy/coordinates/matching.py
--- astropy-0.3.2/astropy/coordinates/matching.py	2014-05-19 11:41:00.520962125 +0200
+++ astropy-0.3.2.six/astropy/coordinates/matching.py	2014-05-19 11:33:25.794283752 +0200
@@ -8,7 +8,7 @@
 
 import numpy as np
 
-from ..extern import six
+import six
 
 __all__ = ['match_coordinates_3d', 'match_coordinates_sky']
 
diff -ur astropy-0.3.2/astropy/coordinates/name_resolve.py astropy-0.3.2.six/astropy/coordinates/name_resolve.py
--- astropy-0.3.2/astropy/coordinates/name_resolve.py	2014-05-19 11:41:00.511962052 +0200
+++ astropy-0.3.2.six/astropy/coordinates/name_resolve.py	2014-05-19 11:33:25.267279498 +0200
@@ -17,7 +17,7 @@
 
 # Astropy
 from ..config import ConfigurationItem
-from ..extern.six.moves import urllib
+from six.moves import urllib
 from .. import units as u
 from .builtin_systems import ICRS
 
diff -ur astropy-0.3.2/astropy/coordinates/tests/test_api.py astropy-0.3.2.six/astropy/coordinates/tests/test_api.py
--- astropy-0.3.2/astropy/coordinates/tests/test_api.py	2014-05-19 11:41:00.514962076 +0200
+++ astropy-0.3.2.six/astropy/coordinates/tests/test_api.py	2014-05-19 11:33:25.405280612 +0200
@@ -7,7 +7,7 @@
 from ...tests.helper import pytest
 raises = pytest.raises
 
-from ...extern import six
+import six
 
 from ... import units as u
 from ..errors import ConvertError, IllegalSecondError, IllegalMinuteError, IllegalHourError
diff -ur astropy-0.3.2/astropy/coordinates/tests/test_arrays.py astropy-0.3.2.six/astropy/coordinates/tests/test_arrays.py
--- astropy-0.3.2/astropy/coordinates/tests/test_arrays.py	2014-05-19 11:41:00.515962084 +0200
+++ astropy-0.3.2.six/astropy/coordinates/tests/test_arrays.py	2014-05-19 11:33:25.455281016 +0200
@@ -3,7 +3,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern import six
+import six
 from ...tests.helper import pytest
 
 import numpy as np
diff -ur astropy-0.3.2/astropy/coordinates/tests/test_formatting.py astropy-0.3.2.six/astropy/coordinates/tests/test_formatting.py
--- astropy-0.3.2/astropy/coordinates/tests/test_formatting.py	2014-05-19 11:41:00.512962060 +0200
+++ astropy-0.3.2.six/astropy/coordinates/tests/test_formatting.py	2014-05-19 11:33:25.346280136 +0200
@@ -1,7 +1,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern import six
+import six
 from ...tests.helper import pytest
 
 from ..angles import Angle
diff -ur astropy-0.3.2/astropy/coordinates/tests/test_name_resolve.py astropy-0.3.2.six/astropy/coordinates/tests/test_name_resolve.py
--- astropy-0.3.2/astropy/coordinates/tests/test_name_resolve.py	2014-05-19 11:41:00.519962117 +0200
+++ astropy-0.3.2.six/astropy/coordinates/tests/test_name_resolve.py	2014-05-19 11:33:25.716283123 +0200
@@ -14,7 +14,7 @@
 from ..name_resolve import (get_icrs_coordinates, NameResolveError,
                             SESAME_DATABASE, _parse_response)
 from ..builtin_systems import ICRS
-from ...extern.six.moves import urllib
+from six.moves import urllib
 from ...tests.helper import remote_data, pytest
 from ... import units as u
 
diff -ur astropy-0.3.2/astropy/coordinates/tests/test_pickle.py astropy-0.3.2.six/astropy/coordinates/tests/test_pickle.py
--- astropy-0.3.2/astropy/coordinates/tests/test_pickle.py	2014-05-19 11:41:00.515962084 +0200
+++ astropy-0.3.2.six/astropy/coordinates/tests/test_pickle.py	2014-05-19 11:33:25.472281153 +0200
@@ -1,4 +1,4 @@
-from ...extern.six.moves import cPickle as pickle
+from six.moves import cPickle as pickle
 from ...coordinates import Longitude
 
 def test_basic():
diff -ur astropy-0.3.2/astropy/coordinates/transformations.py astropy-0.3.2.six/astropy/coordinates/transformations.py
--- astropy-0.3.2/astropy/coordinates/transformations.py	2014-05-19 11:41:00.521962133 +0200
+++ astropy-0.3.2.six/astropy/coordinates/transformations.py	2014-05-19 11:33:25.838284107 +0200
@@ -11,7 +11,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ..extern import six
+import six
 
 import heapq
 import inspect
diff -ur astropy-0.3.2/astropy/io/fits/card.py astropy-0.3.2.six/astropy/io/fits/card.py
--- astropy-0.3.2/astropy/io/fits/card.py	2014-05-19 11:41:00.449961549 +0200
+++ astropy-0.3.2.six/astropy/io/fits/card.py	2014-05-19 11:33:23.253263242 +0200
@@ -11,7 +11,7 @@
 
 from . import ENABLE_RECORD_VALUED_KEYWORD_CARDS, STRIP_HEADER_WHITESPACE
 
-from ...extern.six import string_types, text_type
+from six import string_types, text_type
 from ...utils import deprecated
 from ...utils.exceptions import AstropyUserWarning, AstropyDeprecationWarning
 
diff -ur astropy-0.3.2/astropy/io/fits/diff.py astropy-0.3.2.six/astropy/io/fits/diff.py
--- astropy-0.3.2/astropy/io/fits/diff.py	2014-05-19 11:41:00.471961728 +0200
+++ astropy-0.3.2.six/astropy/io/fits/diff.py	2014-05-19 11:33:23.860268141 +0200
@@ -21,7 +21,7 @@
 import numpy as np
 
 from ... import __version__
-from ...extern.six.moves import reduce
+from six.moves import reduce
 from ...utils import indent
 from .card import Card, BLANK_CARD
 from .header import Header
diff -ur astropy-0.3.2/astropy/io/fits/file.py astropy-0.3.2.six/astropy/io/fits/file.py
--- astropy-0.3.2/astropy/io/fits/file.py	2014-05-19 11:41:00.493961906 +0200
+++ astropy-0.3.2.six/astropy/io/fits/file.py	2014-05-19 11:33:24.610274195 +0200
@@ -16,8 +16,8 @@
 from .util import (isreadable, iswritable, isfile, fileobj_open, fileobj_name,
                    fileobj_closed, fileobj_mode, _array_from_file,
                    _array_to_file, _write_string, encode_ascii)
-from ...extern.six import PY3, b, string_types
-from ...extern.six.moves import urllib
+from six import PY3, b, string_types
+from six.moves import urllib
 from ...utils.exceptions import AstropyUserWarning
 
 
diff -ur astropy-0.3.2/astropy/io/fits/fitsrec.py astropy-0.3.2.six/astropy/io/fits/fitsrec.py
--- astropy-0.3.2/astropy/io/fits/fitsrec.py	2014-05-19 11:41:00.492961898 +0200
+++ astropy-0.3.2.six/astropy/io/fits/fitsrec.py	2014-05-19 11:33:24.555273751 +0200
@@ -13,8 +13,8 @@
                      _AsciiColDefs, _FormatX, _FormatP, _VLF, _get_index,
                      _wrapx, _unwrapx, _makep, _convert_ascii_format, Delayed)
 from .util import decode_ascii, encode_ascii
-from ...extern.six import string_types
-from ...extern.six.moves import xrange, map
+from six import string_types
+from six.moves import xrange, map
 from ...utils import lazyproperty
 from ...utils.exceptions import AstropyDeprecationWarning, AstropyUserWarning
 
diff -ur astropy-0.3.2/astropy/io/fits/hdu/compressed.py astropy-0.3.2.six/astropy/io/fits/hdu/compressed.py
--- astropy-0.3.2/astropy/io/fits/hdu/compressed.py	2014-05-19 11:41:00.459961630 +0200
+++ astropy-0.3.2.six/astropy/io/fits/hdu/compressed.py	2014-05-19 11:33:23.553265664 +0200
@@ -18,8 +18,8 @@
 from ..header import Header
 from ..util import _is_pseudo_unsigned, _unsigned_zero, _is_int
 
-from ....extern.six import string_types, iteritems
-from ....extern.six.moves import xrange
+from six import string_types, iteritems
+from six.moves import xrange
 from ....utils import lazyproperty, deprecated
 from ....utils.exceptions import (AstropyPendingDeprecationWarning,
                                   AstropyUserWarning)
diff -ur astropy-0.3.2/astropy/io/fits/hdu/hdulist.py astropy-0.3.2.six/astropy/io/fits/hdu/hdulist.py
--- astropy-0.3.2/astropy/io/fits/hdu/hdulist.py	2014-05-19 11:41:00.468961703 +0200
+++ astropy-0.3.2.six/astropy/io/fits/hdu/hdulist.py	2014-05-19 11:33:23.803267681 +0200
@@ -18,7 +18,7 @@
                     _get_array_mmap)
 from ..verify import _Verify, _ErrList, VerifyError, VerifyWarning
 
-from ....extern.six import string_types
+from six import string_types
 
 from ....utils import indent
 from ....utils.exceptions import AstropyUserWarning
diff -ur astropy-0.3.2/astropy/io/fits/header.py astropy-0.3.2.six/astropy/io/fits/header.py
--- astropy-0.3.2/astropy/io/fits/header.py	2014-05-19 11:41:00.499961955 +0200
+++ astropy-0.3.2.six/astropy/io/fits/header.py	2014-05-19 11:33:24.786275616 +0200
@@ -17,9 +17,9 @@
 from .util import (encode_ascii, decode_ascii, fileobj_mode, fileobj_closed,
                    fileobj_is_binary)
 
-from ...extern import six
-from ...extern.six import PY3, string_types, itervalues, iteritems, next
-from ...extern.six.moves import zip, range, zip_longest
+import six
+from six import PY3, string_types, itervalues, iteritems, next
+from six.moves import zip, range, zip_longest
 from ...utils import deprecated, isiterable
 from ...utils.exceptions import AstropyUserWarning, AstropyDeprecationWarning
 
diff -ur astropy-0.3.2/astropy/io/fits/tests/test_header.py astropy-0.3.2.six/astropy/io/fits/tests/test_header.py
--- astropy-0.3.2/astropy/io/fits/tests/test_header.py	2014-05-19 11:41:00.481961809 +0200
+++ astropy-0.3.2.six/astropy/io/fits/tests/test_header.py	2014-05-19 11:33:24.210270967 +0200
@@ -11,7 +11,7 @@
 
 from ....io import fits
 from ....io.fits.verify import VerifyWarning
-from ....extern.six import u
+from six import u
 from ....tests.helper import pytest, catch_warnings
 
 from . import FitsTestCase
diff -ur astropy-0.3.2/astropy/io/fits/tests/test_table.py astropy-0.3.2.six/astropy/io/fits/tests/test_table.py
--- astropy-0.3.2/astropy/io/fits/tests/test_table.py	2014-05-19 11:41:00.487961858 +0200
+++ astropy-0.3.2.six/astropy/io/fits/tests/test_table.py	2014-05-19 11:33:24.428272726 +0200
@@ -3,8 +3,8 @@
 import numpy as np
 from numpy import char as chararray
 
-from ....extern.six.moves import range
-from ....extern.six.moves import cPickle as pickle
+from six.moves import range
+from six.moves import cPickle as pickle
 from ....io import fits
 from ....tests.helper import pytest
 
diff -ur astropy-0.3.2/astropy/io/votable/connect.py astropy-0.3.2.six/astropy/io/votable/connect.py
--- astropy-0.3.2/astropy/io/votable/connect.py	2014-05-19 11:41:00.432961412 +0200
+++ astropy-0.3.2.six/astropy/io/votable/connect.py	2014-05-19 11:33:22.646258343 +0200
@@ -4,7 +4,7 @@
 
 import os
 
-from ...extern import six
+import six
 
 from . import parse, from_table
 from .tree import VOTableFile, Table as VOTable
diff -ur astropy-0.3.2/astropy/io/votable/converters.py astropy-0.3.2.six/astropy/io/votable/converters.py
--- astropy-0.3.2/astropy/io/votable/converters.py	2014-05-19 11:41:00.429961387 +0200
+++ astropy-0.3.2.six/astropy/io/votable/converters.py	2014-05-19 11:33:22.561257656 +0200
@@ -5,8 +5,8 @@
 """
 
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ...extern import six
-from ...extern.six.moves import xrange
+import six
+from six.moves import xrange
 
 # STDLIB
 import re
diff -ur astropy-0.3.2/astropy/io/votable/exceptions.py astropy-0.3.2.six/astropy/io/votable/exceptions.py
--- astropy-0.3.2/astropy/io/votable/exceptions.py	2014-05-19 11:41:00.431961403 +0200
+++ astropy-0.3.2.six/astropy/io/votable/exceptions.py	2014-05-19 11:33:22.624258165 +0200
@@ -34,7 +34,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern import six
+import six
 
 # STDLIB
 import io
diff -ur astropy-0.3.2/astropy/io/votable/table.py astropy-0.3.2.six/astropy/io/votable/table.py
--- astropy-0.3.2/astropy/io/votable/table.py	2014-05-19 11:41:00.433961420 +0200
+++ astropy-0.3.2.six/astropy/io/votable/table.py	2014-05-19 11:33:22.699258770 +0200
@@ -8,7 +8,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern import six
+import six
 
 # STDLIB
 import io
diff -ur astropy-0.3.2/astropy/io/votable/tests/ucd_test.py astropy-0.3.2.six/astropy/io/votable/tests/ucd_test.py
--- astropy-0.3.2/astropy/io/votable/tests/ucd_test.py	2014-05-19 11:41:00.437961452 +0200
+++ astropy-0.3.2.six/astropy/io/votable/tests/ucd_test.py	2014-05-19 11:33:22.891260320 +0200
@@ -1,7 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ....extern import six
+import six
 
 
 from ....tests.helper import raises
diff -ur astropy-0.3.2/astropy/io/votable/tests/vo_test.py astropy-0.3.2.six/astropy/io/votable/tests/vo_test.py
--- astropy-0.3.2/astropy/io/votable/tests/vo_test.py	2014-05-19 11:41:00.435961436 +0200
+++ astropy-0.3.2.six/astropy/io/votable/tests/vo_test.py	2014-05-19 11:33:22.804259618 +0200
@@ -4,8 +4,8 @@
 """
 
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ....extern import six
-from ....extern.six.moves import xrange
+import six
+from six.moves import xrange
 
 # STDLIB
 import difflib
diff -ur astropy-0.3.2/astropy/io/votable/tree.py astropy-0.3.2.six/astropy/io/votable/tree.py
--- astropy-0.3.2/astropy/io/votable/tree.py	2014-05-19 11:41:00.443961501 +0200
+++ astropy-0.3.2.six/astropy/io/votable/tree.py	2014-05-19 11:33:23.074261797 +0200
@@ -2,8 +2,8 @@
 # TODO: Test FITS parsing
 
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ...extern import six
-from ...extern.six.moves import xrange, urllib
+import six
+from six.moves import xrange, urllib
 
 # STDLIB
 import base64
diff -ur astropy-0.3.2/astropy/io/votable/util.py astropy-0.3.2.six/astropy/io/votable/util.py
--- astropy-0.3.2/astropy/io/votable/util.py	2014-05-19 11:41:00.444961509 +0200
+++ astropy-0.3.2.six/astropy/io/votable/util.py	2014-05-19 11:33:23.118262152 +0200
@@ -4,7 +4,7 @@
 """
 
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ...extern import six
+import six
 
 # STDLIB
 import codecs
diff -ur astropy-0.3.2/astropy/io/votable/validator/html.py astropy-0.3.2.six/astropy/io/votable/validator/html.py
--- astropy-0.3.2/astropy/io/votable/validator/html.py	2014-05-19 11:41:00.425961355 +0200
+++ astropy-0.3.2.six/astropy/io/votable/validator/html.py	2014-05-19 11:33:22.440256680 +0200
@@ -1,7 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ....extern import six
-from ....extern.six.moves import xrange
+import six
+from six.moves import xrange
 
 # STDLIB
 import contextlib
diff -ur astropy-0.3.2/astropy/io/votable/validator/main.py astropy-0.3.2.six/astropy/io/votable/validator/main.py
--- astropy-0.3.2/astropy/io/votable/validator/main.py	2014-05-19 11:41:00.426961363 +0200
+++ astropy-0.3.2.six/astropy/io/votable/validator/main.py	2014-05-19 11:33:22.463256866 +0200
@@ -4,7 +4,7 @@
 and generates a report as a directory tree of HTML files.
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ....extern import six
+import six
 
 # STDLIB
 import os
diff -ur astropy-0.3.2/astropy/io/votable/validator/result.py astropy-0.3.2.six/astropy/io/votable/validator/result.py
--- astropy-0.3.2/astropy/io/votable/validator/result.py	2014-05-19 11:41:00.425961355 +0200
+++ astropy-0.3.2.six/astropy/io/votable/validator/result.py	2014-05-19 11:33:22.413256462 +0200
@@ -5,9 +5,9 @@
 """
 
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ....extern import six
-from ....extern.six.moves import http_client, urllib
-from ....extern.six.moves import cPickle as pickle
+import six
+from six.moves import http_client, urllib
+from six.moves import cPickle as pickle
 
 # STDLIB
 from xml.parsers.expat import ExpatError
diff -ur astropy-0.3.2/astropy/setup_helpers.py astropy-0.3.2.six/astropy/setup_helpers.py
--- astropy-0.3.2/astropy/setup_helpers.py	2014-05-19 11:41:00.524962157 +0200
+++ astropy-0.3.2.six/astropy/setup_helpers.py	2014-05-19 11:33:25.906284656 +0200
@@ -40,7 +40,7 @@
 from .utils.misc import walk_skip_hidden
 from .utils.exceptions import AstropyDeprecationWarning
 
-from .extern import six
+import six
 
 
 try:
diff -ur astropy-0.3.2/astropy/stats/funcs.py astropy-0.3.2.six/astropy/stats/funcs.py
--- astropy-0.3.2/astropy/stats/funcs.py	2014-05-19 11:41:00.361960836 +0200
+++ astropy-0.3.2.six/astropy/stats/funcs.py	2014-05-19 11:33:17.617217752 +0200
@@ -13,7 +13,7 @@
 
 import numpy as np
 
-from ..extern.six.moves import xrange
+from six.moves import xrange
 
 
 __all__ = ['sigma_clip', 'binom_conf_interval', 'binned_binom_proportion',
diff -ur astropy-0.3.2/astropy/table/groups.py astropy-0.3.2.six/astropy/table/groups.py
--- astropy-0.3.2/astropy/table/groups.py	2014-05-19 11:41:00.364960860 +0200
+++ astropy-0.3.2.six/astropy/table/groups.py	2014-05-19 11:33:17.731218672 +0200
@@ -1,7 +1,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
-from ..extern.six.moves import zip as izip
+import six
+from six.moves import zip as izip
 
 import platform
 import warnings
diff -ur astropy-0.3.2/astropy/table/__init__.py astropy-0.3.2.six/astropy/table/__init__.py
--- astropy-0.3.2/astropy/table/__init__.py	2014-05-19 11:41:00.370960909 +0200
+++ astropy-0.3.2.six/astropy/table/__init__.py	2014-05-19 11:33:17.897220011 +0200
@@ -1,7 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
+import six
 
 from .table import Column, Table, TableColumns, Row, MaskedColumn
 from .np_utils import TableMergeError
diff -ur astropy-0.3.2/astropy/table/np_utils.py astropy-0.3.2.six/astropy/table/np_utils.py
--- astropy-0.3.2/astropy/table/np_utils.py	2014-05-19 11:41:00.364960860 +0200
+++ astropy-0.3.2.six/astropy/table/np_utils.py	2014-05-19 11:33:17.702218438 +0200
@@ -10,8 +10,8 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
-from ..extern.six.moves import zip as izip
+import six
+from six.moves import zip as izip
 
 from itertools import chain
 import collections
diff -ur astropy-0.3.2/astropy/table/operations.py astropy-0.3.2.six/astropy/table/operations.py
--- astropy-0.3.2/astropy/table/operations.py	2014-05-19 11:41:00.370960909 +0200
+++ astropy-0.3.2.six/astropy/table/operations.py	2014-05-19 11:33:17.857219689 +0200
@@ -8,7 +8,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
+import six
 
 from copy import deepcopy
 import warnings
diff -ur astropy-0.3.2/astropy/table/pprint.py astropy-0.3.2.six/astropy/table/pprint.py
--- astropy-0.3.2/astropy/table/pprint.py	2014-05-19 11:41:00.363960852 +0200
+++ astropy-0.3.2.six/astropy/table/pprint.py	2014-05-19 11:33:17.667218155 +0200
@@ -1,10 +1,10 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
-from ..extern.six import text_type
-from ..extern.six.moves import zip as izip
-from ..extern.six.moves import xrange
+import six
+from six import text_type
+from six.moves import zip as izip
+from six.moves import xrange
 
 import os
 import sys
diff -ur astropy-0.3.2/astropy/table/table.py astropy-0.3.2.six/astropy/table/table.py
--- astropy-0.3.2/astropy/table/table.py	2014-05-19 11:41:00.369960901 +0200
+++ astropy-0.3.2.six/astropy/table/table.py	2014-05-19 11:33:17.830219471 +0200
@@ -2,9 +2,9 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ..extern import six
-from ..extern.six.moves import zip as izip
-from ..extern.six.moves import range as xrange
+import six
+from six.moves import zip as izip
+from six.moves import range as xrange
 
 import abc
 import collections
diff -ur astropy-0.3.2/astropy/table/tests/test_pickle.py astropy-0.3.2.six/astropy/table/tests/test_pickle.py
--- astropy-0.3.2/astropy/table/tests/test_pickle.py	2014-05-19 11:41:00.376960958 +0200
+++ astropy-0.3.2.six/astropy/table/tests/test_pickle.py	2014-05-19 11:33:18.137221949 +0200
@@ -1,4 +1,4 @@
-from ...extern.six.moves import cPickle as pickle
+from six.moves import cPickle as pickle
 
 import numpy as np
 import pytest
diff -ur astropy-0.3.2/astropy/table/tests/test_pprint.py astropy-0.3.2.six/astropy/table/tests/test_pprint.py
--- astropy-0.3.2/astropy/table/tests/test_pprint.py	2014-05-19 11:41:00.375960950 +0200
+++ astropy-0.3.2.six/astropy/table/tests/test_pprint.py	2014-05-19 11:33:18.088221553 +0200
@@ -5,7 +5,7 @@
 from ...tests.helper import pytest
 from ... import table
 from ...table import pprint
-from ...extern.six import PY3
+from six import PY3
 
 BIG_WIDE_ARR = np.arange(2000, dtype=np.float).reshape(100, 20)
 SMALL_ARR = np.arange(12, dtype=np.int).reshape(4, 3)
diff -ur astropy-0.3.2/astropy/table/tests/test_table.py astropy-0.3.2.six/astropy/table/tests/test_table.py
--- astropy-0.3.2/astropy/table/tests/test_table.py	2014-05-19 11:41:00.379960982 +0200
+++ astropy-0.3.2.six/astropy/table/tests/test_table.py	2014-05-19 11:33:18.239222772 +0200
@@ -9,7 +9,7 @@
 
 import numpy as np
 
-from ...extern import six
+import six
 from ...tests.helper import pytest, assert_follows_unicode_guidelines
 from ... import table
 from ... import units as u
diff -ur astropy-0.3.2/astropy/tests/helper.py astropy-0.3.2.six/astropy/tests/helper.py
--- astropy-0.3.2/astropy/tests/helper.py	2014-05-19 11:41:00.622962952 +0200
+++ astropy-0.3.2.six/astropy/tests/helper.py	2014-05-19 11:33:31.407329061 +0200
@@ -6,8 +6,8 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ..extern import six
-from ..extern.six.moves import cPickle as pickle
+import six
+from six.moves import cPickle as pickle
 
 import errno
 import shlex
@@ -546,7 +546,7 @@
         If not provided, no roundtrip testing will be performed.
     """
     from .. import UNICODE_OUTPUT
-    from ..extern import six
+    import six
 
     UNICODE_OUTPUT.set(False)
 
diff -ur astropy-0.3.2/astropy/tests/pytest_plugins.py astropy-0.3.2.six/astropy/tests/pytest_plugins.py
--- astropy-0.3.2/astropy/tests/pytest_plugins.py	2014-05-19 11:41:00.626962984 +0200
+++ astropy-0.3.2.six/astropy/tests/pytest_plugins.py	2014-05-19 11:33:31.619330772 +0200
@@ -5,8 +5,8 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
-from ..extern.six.moves import filter
+import six
+from six.moves import filter
 
 import doctest
 import fnmatch
diff -ur astropy-0.3.2/astropy/tests/tests/test_imports.py astropy-0.3.2.six/astropy/tests/tests/test_imports.py
--- astropy-0.3.2/astropy/tests/tests/test_imports.py	2014-05-19 11:41:00.624962968 +0200
+++ astropy-0.3.2.six/astropy/tests/tests/test_imports.py	2014-05-19 11:33:31.548330199 +0200
@@ -1,7 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ...extern import six
+import six
 
 import pkgutil
 import os
diff -ur astropy-0.3.2/astropy/time/core.py astropy-0.3.2.six/astropy/time/core.py
--- astropy-0.3.2/astropy/time/core.py	2014-05-19 11:41:00.632963033 +0200
+++ astropy-0.3.2.six/astropy/time/core.py	2014-05-19 11:33:31.857332693 +0200
@@ -20,7 +20,7 @@
 from .. import units as u
 from ..utils import deprecated, deprecated_attribute
 from ..utils.compat.misc import override__dir__
-from ..extern import six
+import six
 
 __all__ = ['Time', 'TimeDelta', 'TimeFormat', 'TimeJD', 'TimeMJD',
            'TimeFromEpoch', 'TimeUnix', 'TimeCxcSec', 'TimeGPS', 'TimePlotDate',
diff -ur astropy-0.3.2/astropy/time/tests/test_pickle.py astropy-0.3.2.six/astropy/time/tests/test_pickle.py
--- astropy-0.3.2/astropy/time/tests/test_pickle.py	2014-05-19 11:41:00.635963057 +0200
+++ astropy-0.3.2.six/astropy/time/tests/test_pickle.py	2014-05-19 11:33:32.005333888 +0200
@@ -3,7 +3,7 @@
 import numpy as np
 
 from .. import Time
-from ...extern.six.moves import cPickle as pickle
+from six.moves import cPickle as pickle
 
 
 class TestPickle():
diff -ur astropy-0.3.2/astropy/units/core.py astropy-0.3.2.six/astropy/units/core.py
--- astropy-0.3.2/astropy/units/core.py	2014-05-19 11:41:00.604962806 +0200
+++ astropy-0.3.2.six/astropy/units/core.py	2014-05-19 11:33:30.099318502 +0200
@@ -7,8 +7,8 @@
 
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
-from ..extern.six.moves import zip
+import six
+from six.moves import zip
 
 import inspect
 import sys
diff -ur astropy-0.3.2/astropy/units/format/cds.py astropy-0.3.2.six/astropy/units/format/cds.py
--- astropy-0.3.2/astropy/units/format/cds.py	2014-05-19 11:41:00.595962733 +0200
+++ astropy-0.3.2.six/astropy/units/format/cds.py	2014-05-19 11:33:29.731315532 +0200
@@ -12,8 +12,8 @@
 import os
 import re
 
-from ...extern import six
-from ...extern.six.moves import zip
+import six
+from six.moves import zip
 
 from .base import Base
 from . import utils
diff -ur astropy-0.3.2/astropy/units/format/fits.py astropy-0.3.2.six/astropy/units/format/fits.py
--- astropy-0.3.2/astropy/units/format/fits.py	2014-05-19 11:41:00.595962733 +0200
+++ astropy-0.3.2.six/astropy/units/format/fits.py	2014-05-19 11:33:29.753315709 +0200
@@ -6,7 +6,7 @@
 
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ...extern.six.moves import zip
+from six.moves import zip
 
 import keyword
 import warnings
diff -ur astropy-0.3.2/astropy/units/format/generic.py astropy-0.3.2.six/astropy/units/format/generic.py
--- astropy-0.3.2/astropy/units/format/generic.py	2014-05-19 11:41:00.597962749 +0200
+++ astropy-0.3.2.six/astropy/units/format/generic.py	2014-05-19 11:33:29.855316533 +0200
@@ -7,7 +7,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern import six
+import six
 
 import os
 import re
diff -ur astropy-0.3.2/astropy/units/format/utils.py astropy-0.3.2.six/astropy/units/format/utils.py
--- astropy-0.3.2/astropy/units/format/utils.py	2014-05-19 11:41:00.593962717 +0200
+++ astropy-0.3.2.six/astropy/units/format/utils.py	2014-05-19 11:33:29.645314838 +0200
@@ -8,7 +8,7 @@
                         unicode_literals)
 
 
-from ...extern import six
+import six
 from ...utils.compat.fractions import Fraction
 
 
diff -ur astropy-0.3.2/astropy/units/format/vounit.py astropy-0.3.2.six/astropy/units/format/vounit.py
--- astropy-0.3.2/astropy/units/format/vounit.py	2014-05-19 11:41:00.594962725 +0200
+++ astropy-0.3.2.six/astropy/units/format/vounit.py	2014-05-19 11:33:29.685315160 +0200
@@ -4,8 +4,8 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ...extern import six
-from ...extern.six.moves import zip
+import six
+from six.moves import zip
 
 import keyword
 import warnings
diff -ur astropy-0.3.2/astropy/units/quantity.py astropy-0.3.2.six/astropy/units/quantity.py
--- astropy-0.3.2/astropy/units/quantity.py	2014-05-19 11:41:00.593962717 +0200
+++ astropy-0.3.2.six/astropy/units/quantity.py	2014-05-19 11:33:29.600314474 +0200
@@ -15,7 +15,7 @@
 import numpy as np
 
 # AstroPy
-from ..extern import six
+import six
 from .core import (Unit, dimensionless_unscaled, UnitBase, UnitsError,
                    get_current_unit_registry)
 from ..utils import lazyproperty
diff -ur astropy-0.3.2/astropy/units/tests/test_equivalencies.py astropy-0.3.2.six/astropy/units/tests/test_equivalencies.py
--- astropy-0.3.2/astropy/units/tests/test_equivalencies.py	2014-05-19 11:41:00.608962838 +0200
+++ astropy-0.3.2.six/astropy/units/tests/test_equivalencies.py	2014-05-19 11:33:30.375320730 +0200
@@ -12,7 +12,7 @@
 
 from ...tests.helper import pytest
 
-from ...extern.six.moves import zip
+from six.moves import zip
 from ... import units as u
 
 
diff -ur astropy-0.3.2/astropy/units/tests/test_quantity.py astropy-0.3.2.six/astropy/units/tests/test_quantity.py
--- astropy-0.3.2/astropy/units/tests/test_quantity.py	2014-05-19 11:41:00.611962862 +0200
+++ astropy-0.3.2.six/astropy/units/tests/test_quantity.py	2014-05-19 11:33:30.465321457 +0200
@@ -17,9 +17,9 @@
 from ...utils import isiterable
 from ... import units as u
 from ...units.quantity import _UNIT_NOT_INITIALISED
-from ...extern.six.moves import xrange
-from ...extern.six.moves import cPickle as pickle
-from ...extern import six
+from six.moves import xrange
+from six.moves import cPickle as pickle
+import six
 
 """ The Quantity class will represent a number + unit + uncertainty """
 
diff -ur astropy-0.3.2/astropy/units/tests/test_units.py astropy-0.3.2.six/astropy/units/tests/test_units.py
--- astropy-0.3.2/astropy/units/tests/test_units.py	2014-05-19 11:41:00.606962822 +0200
+++ astropy-0.3.2.six/astropy/units/tests/test_units.py	2014-05-19 11:33:30.188319221 +0200
@@ -12,8 +12,8 @@
 import numpy as np
 from numpy.testing.utils import assert_allclose
 
-from ...extern import six
-from ...extern.six.moves import cPickle as pickle
+import six
+from six.moves import cPickle as pickle
 from ...tests.helper import pytest, raises, catch_warnings
 from ...utils.compat.fractions import Fraction
 
diff -ur astropy-0.3.2/astropy/units/utils.py astropy-0.3.2.six/astropy/units/utils.py
--- astropy-0.3.2/astropy/units/utils.py	2014-05-19 11:41:00.590962692 +0200
+++ astropy-0.3.2.six/astropy/units/utils.py	2014-05-19 11:33:29.516313796 +0200
@@ -18,7 +18,7 @@
 import numpy as np
 from numpy import finfo
 
-from ..extern import six
+import six
 from ..utils.compat.fractions import Fraction
 from ..utils.exceptions import AstropyDeprecationWarning
 
diff -ur astropy-0.3.2/astropy/utils/compat/misc.py astropy-0.3.2.six/astropy/utils/compat/misc.py
--- astropy-0.3.2/astropy/utils/compat/misc.py	2014-05-19 11:41:00.568962514 +0200
+++ astropy-0.3.2.six/astropy/utils/compat/misc.py	2014-05-19 11:33:28.284303851 +0200
@@ -16,7 +16,7 @@
 
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ...extern import six
+import six
 
 import os
 import sys
diff -ur astropy-0.3.2/astropy/utils/console.py astropy-0.3.2.six/astropy/utils/console.py
--- astropy-0.3.2/astropy/utils/console.py	2014-05-19 11:41:00.556962417 +0200
+++ astropy-0.3.2.six/astropy/utils/console.py	2014-05-19 11:33:27.146294665 +0200
@@ -58,8 +58,8 @@
     _HAVE_IPYTHON = False
 
 from ..config import ConfigurationItem
-from ..extern import six
-from ..extern.six.moves import range
+import six
+from six.moves import range
 
 from .misc import deprecated, isiterable
 
diff -ur astropy-0.3.2/astropy/utils/data.py astropy-0.3.2.six/astropy/utils/data.py
--- astropy-0.3.2/astropy/utils/data.py	2014-05-19 11:41:00.576962579 +0200
+++ astropy-0.3.2.six/astropy/utils/data.py	2014-05-19 11:33:28.555306039 +0200
@@ -7,8 +7,8 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ..extern import six
-from ..extern.six.moves import urllib
+import six
+from six.moves import urllib
 
 import atexit
 import contextlib
diff -ur astropy-0.3.2/astropy/utils/metadata.py astropy-0.3.2.six/astropy/utils/metadata.py
--- astropy-0.3.2/astropy/utils/metadata.py	2014-05-19 11:41:00.557962425 +0200
+++ astropy-0.3.2.six/astropy/utils/metadata.py	2014-05-19 11:33:27.167294835 +0200
@@ -4,7 +4,7 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ..extern import six
+import six
 
 import warnings
 
diff -ur astropy-0.3.2/astropy/utils/misc.py astropy-0.3.2.six/astropy/utils/misc.py
--- astropy-0.3.2/astropy/utils/misc.py	2014-05-19 11:41:00.574962562 +0200
+++ astropy-0.3.2.six/astropy/utils/misc.py	2014-05-19 11:33:28.500305595 +0200
@@ -24,8 +24,8 @@
 
 from .exceptions import AstropyDeprecationWarning, AstropyPendingDeprecationWarning
 
-from ..extern import six
-from ..extern.six.moves import urllib
+import six
+from six.moves import urllib
 
 
 __all__ = ['find_current_module', 'isiterable', 'deprecated', 'lazyproperty',
diff -ur astropy-0.3.2/astropy/utils/tests/test_console.py astropy-0.3.2.six/astropy/utils/tests/test_console.py
--- astropy-0.3.2/astropy/utils/tests/test_console.py	2014-05-19 11:41:00.581962619 +0200
+++ astropy-0.3.2.six/astropy/utils/tests/test_console.py	2014-05-19 11:33:28.854308452 +0200
@@ -3,8 +3,8 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern.six import next
-from ...extern.six.moves import xrange
+from six import next
+from six.moves import xrange
 
 import io
 import locale
diff -ur astropy-0.3.2/astropy/utils/tests/test_data.py astropy-0.3.2.six/astropy/utils/tests/test_data.py
--- astropy-0.3.2/astropy/utils/tests/test_data.py	2014-05-19 11:41:00.581962619 +0200
+++ astropy-0.3.2.six/astropy/utils/tests/test_data.py	2014-05-19 11:33:28.832308275 +0200
@@ -2,7 +2,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern import six
+import six
 
 from ...tests.helper import remote_data, raises, pytest, catch_warnings
 
@@ -304,7 +304,7 @@
     checks that download_file gives a URLError and not an AttributeError,
     as its code pathway involves some fiddling with the exception.
     """
-    from ...extern.six.moves.urllib_error import URLError
+    from six.moves.urllib_error import URLError
     from ..data import download_file
 
     with pytest.raises(URLError):
diff -ur astropy-0.3.2/astropy/utils/tests/test_odict.py astropy-0.3.2.six/astropy/utils/tests/test_odict.py
--- astropy-0.3.2/astropy/utils/tests/test_odict.py	2014-05-19 11:41:00.578962595 +0200
+++ astropy-0.3.2.six/astropy/utils/tests/test_odict.py	2014-05-19 11:33:28.614306515 +0200
@@ -10,7 +10,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern import six
+import six
 
 import unittest
 import inspect
diff -ur astropy-0.3.2/astropy/utils/tests/test_xml.py astropy-0.3.2.six/astropy/utils/tests/test_xml.py
--- astropy-0.3.2/astropy/utils/tests/test_xml.py	2014-05-19 11:41:00.579962603 +0200
+++ astropy-0.3.2.six/astropy/utils/tests/test_xml.py	2014-05-19 11:33:28.715307330 +0200
@@ -2,7 +2,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern import six
+import six
 
 import io
 
diff -ur astropy-0.3.2/astropy/utils/timer.py astropy-0.3.2.six/astropy/utils/timer.py
--- astropy-0.3.2/astropy/utils/timer.py	2014-05-19 11:41:00.559962441 +0200
+++ astropy-0.3.2.six/astropy/utils/timer.py	2014-05-19 11:33:27.314296022 +0200
@@ -4,8 +4,8 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ..extern import six
-from ..extern.six.moves import xrange, map
+import six
+from six.moves import xrange, map
 
 # STDLIB
 import time
diff -ur astropy-0.3.2/astropy/utils/xml/check.py astropy-0.3.2.six/astropy/utils/xml/check.py
--- astropy-0.3.2/astropy/utils/xml/check.py	2014-05-19 11:41:00.560962449 +0200
+++ astropy-0.3.2.six/astropy/utils/xml/check.py	2014-05-19 11:33:27.333296175 +0200
@@ -6,7 +6,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from ...extern.six.moves import xrange, urllib
+from six.moves import xrange, urllib
 
 import re
 
diff -ur astropy-0.3.2/astropy/utils/xml/iterparser.py astropy-0.3.2.six/astropy/utils/xml/iterparser.py
--- astropy-0.3.2/astropy/utils/xml/iterparser.py	2014-05-19 11:41:00.561962457 +0200
+++ astropy-0.3.2.six/astropy/utils/xml/iterparser.py	2014-05-19 11:33:27.397296691 +0200
@@ -4,7 +4,7 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ...extern import six
+import six
 
 # STDLIB
 import collections
diff -ur astropy-0.3.2/astropy/utils/xml/writer.py astropy-0.3.2.six/astropy/utils/xml/writer.py
--- astropy-0.3.2/astropy/utils/xml/writer.py	2014-05-19 11:41:00.562962465 +0200
+++ astropy-0.3.2.six/astropy/utils/xml/writer.py	2014-05-19 11:33:27.457297176 +0200
@@ -5,7 +5,7 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from ...extern import six
+import six
 
 # STDLIB
 import contextlib
diff -ur astropy-0.3.2/astropy/wcs/setup_package.py astropy-0.3.2.six/astropy/wcs/setup_package.py
--- astropy-0.3.2/astropy/wcs/setup_package.py	2014-05-19 11:41:00.551962376 +0200
+++ astropy-0.3.2.six/astropy/wcs/setup_package.py	2014-05-19 11:33:27.003293511 +0200
@@ -12,7 +12,7 @@
 import sys
 
 from astropy import setup_helpers
-from astropy.extern import six
+import six
 
 WCSROOT = os.path.relpath(os.path.dirname(__file__))
 WCSVERSION = "4.10"
diff -ur astropy-0.3.2/astropy/wcs/tests/test_pickle.py astropy-0.3.2.six/astropy/wcs/tests/test_pickle.py
--- astropy-0.3.2/astropy/wcs/tests/test_pickle.py	2014-05-19 11:41:00.549962360 +0200
+++ astropy-0.3.2.six/astropy/wcs/tests/test_pickle.py	2014-05-19 11:33:26.919292833 +0200
@@ -6,7 +6,7 @@
 import numpy as np
 from numpy.testing import assert_array_almost_equal
 
-from ...extern.six.moves import cPickle
+from six.moves import cPickle
 from ...utils.data import get_pkg_data_contents, get_pkg_data_fileobj
 from ...utils.misc import NumpyRNGContext
 from ...io import fits
diff -ur astropy-0.3.2/astropy/wcs/tests/test_wcs.py astropy-0.3.2.six/astropy/wcs/tests/test_wcs.py
--- astropy-0.3.2/astropy/wcs/tests/test_wcs.py	2014-05-19 11:41:00.548962352 +0200
+++ astropy-0.3.2.six/astropy/wcs/tests/test_wcs.py	2014-05-19 11:33:26.899292672 +0200
@@ -1,7 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from ...extern import six
+import six
 
 import os
 import sys
diff -ur astropy-0.3.2/astropy/wcs/wcs.py astropy-0.3.2.six/astropy/wcs/wcs.py
--- astropy-0.3.2/astropy/wcs/wcs.py	2014-05-19 11:41:00.555962409 +0200
+++ astropy-0.3.2.six/astropy/wcs/wcs.py	2014-05-19 11:33:27.099294286 +0200
@@ -39,7 +39,7 @@
 import numpy as np
 
 # LOCAL
-from ..extern import six
+import six
 from ..io import fits
 from . import _docutil as __
 try:
