diff -ur astropy-0.4.2/astropy/modeling/tests/test_models.py astropy-0.4.2.2516/astropy/modeling/tests/test_models.py
--- astropy-0.4.2/astropy/modeling/tests/test_models.py	2014-12-05 11:26:51.485613491 +0100
+++ astropy-0.4.2.2516/astropy/modeling/tests/test_models.py	2014-12-05 11:26:29.097428190 +0100
@@ -395,6 +395,7 @@
         y = test_parameters['y_values']
         utils.assert_allclose(model(x), y, atol=self.eval_error)
 
+    @pytest.skip("Skip in Fedora. (github isssue #2516)")
     @pytest.mark.skipif('not HAS_SCIPY')
     def test_fitter1D(self, model_class, test_parameters):
         """
