diff -ur astropy-4.2/astropy/config/configuration.py astropy-4.2.configobj/astropy/config/configuration.py
--- astropy-4.2/astropy/config/configuration.py	2020-11-25 08:28:48.000000000 +0100
+++ astropy-4.2.configobj/astropy/config/configuration.py	2021-02-15 23:12:16.720541628 +0100
@@ -22,7 +22,7 @@
 from warnings import warn
 from contextlib import contextmanager
 
-from astropy.extern.configobj import configobj, validate
+import configobj, validate
 from astropy.utils import find_current_module, silence
 from astropy.utils.exceptions import AstropyDeprecationWarning, AstropyWarning
 from astropy.utils.introspection import resolve_name
