diff -up astropy-1.0.5/astropy/tests/test_logger.py.pytest23 astropy-1.0.5/astropy/tests/test_logger.py
--- astropy-1.0.5/astropy/tests/test_logger.py.pytest23	2015-10-05 15:04:07.000000000 -0600
+++ astropy-1.0.5/astropy/tests/test_logger.py	2015-10-28 13:37:11.742479031 -0600
@@ -261,7 +261,7 @@ def test_exception_logging_origin():
     assert log_list[0].origin == 'astropy.utils.collections'
 
 
-@pytest.mark.skipif(sys.version_info[:2] >= (3, 5),
+@pytest.mark.skipif("sys.version_info[:2] >= (3, 5)",
                     reason="Infinite recursion on Python 3.5")
 @pytest.mark.xfail(str("ip is not None"))
 def test_exception_logging_argless_exception():
