diff -ur astropy-0.3.2/astropy/wcs/tests/test_wcsprm.py astropy-0.3.2.wcslib.423/astropy/wcs/tests/test_wcsprm.py
--- astropy-0.3.2/astropy/wcs/tests/test_wcsprm.py	2014-05-26 11:14:27.023648667 +0200
+++ astropy-0.3.2.wcslib.423/astropy/wcs/tests/test_wcsprm.py	2014-05-26 11:13:49.630346134 +0200
@@ -639,7 +639,7 @@
 
 def test_velangl():
     w = _wcs.Wcsprm()
-    assert w.velangl == 0.0
+    assert np.isnan(w.velangl)
     w.velangl = 42.0
     assert w.velangl == 42.0
     del w.velangl
diff -ur astropy-0.3.2/astropy/wcs/tests/test_wcs.py astropy-0.3.2.wcslib.423/astropy/wcs/tests/test_wcs.py
--- astropy-0.3.2/astropy/wcs/tests/test_wcs.py	2014-05-26 11:14:27.021648651 +0200
+++ astropy-0.3.2.wcslib.423/astropy/wcs/tests/test_wcs.py	2014-05-26 11:14:41.149762955 +0200
@@ -453,7 +453,7 @@
 
 def test_to_header_string():
     header_string = """
-    WCSAXES =                    2 / Number of coordinate axes                      CRPIX1  =                    0 / Pixel coordinate of reference point            CRPIX2  =                    0 / Pixel coordinate of reference point            CDELT1  =                    1 / Coordinate increment at reference point        CDELT2  =                    1 / Coordinate increment at reference point        CRVAL1  =                    0 / Coordinate value at reference point            CRVAL2  =                    0 / Coordinate value at reference point            LATPOLE =                   90 / [deg] Native latitude of celestial pole        RESTFRQ =                    0 / [Hz] Line rest frequency                       RESTWAV =                    0 / [Hz] Line rest wavelength                      END"""
+    WCSAXES =                    2 / Number of coordinate axes                      CRPIX1  =                  0.0 / Pixel coordinate of reference point            CRPIX2  =                  0.0 / Pixel coordinate of reference point            CDELT1  =                  1.0 / Coordinate increment at reference point        CDELT2  =                  1.0 / Coordinate increment at reference point        CRVAL1  =                  0.0 / Coordinate value at reference point            CRVAL2  =                  0.0 / Coordinate value at reference point            LATPOLE =                 90.0 / [deg] Native latitude of celestial pole        END"""
 
     w = wcs.WCS()
     assert w.to_header_string().strip() == header_string.strip()
