diff -ur astropy-1.1.post2/astropy/config/configuration.py astropy-1.1.post2.configobj/astropy/config/configuration.py
--- astropy-1.1.post2/astropy/config/configuration.py	2015-12-11 18:45:19.000000000 +0100
+++ astropy-1.1.post2.configobj/astropy/config/configuration.py	2016-01-06 18:48:19.049244958 +0100
@@ -23,7 +23,8 @@
 import types
 from warnings import warn
 
-from ..extern.configobj import configobj, validate
+import configobj
+import validate
 from ..utils.exceptions import AstropyWarning, AstropyDeprecationWarning
 from ..utils import find_current_module
 from ..utils.introspection import resolve_name
