diff -ur astropy-0.4.1/astropy/tests/helper.py astropy-0.4.1.pytest/astropy/tests/helper.py
--- astropy-0.4.1/astropy/tests/helper.py	2014-09-01 23:06:03.280899166 +0200
+++ astropy-0.4.1.pytest/astropy/tests/helper.py	2014-09-01 23:08:32.232105514 +0200
@@ -37,7 +37,8 @@
 from ..utils.exceptions import AstropyWarning
 from ..config import configuration
 
-if os.environ.get('ASTROPY_USE_SYSTEM_PYTEST') or '_pytest' in sys.modules:
+# Use system pytest always, modified in the Fedora package
+if True or os.environ.get('ASTROPY_USE_SYSTEM_PYTEST') or '_pytest' in sys.modules:
     import pytest
 
 else:
