diff -ur astropy-5.2/astropy/config/configuration.py astropy-5.2.config/astropy/config/configuration.py
--- astropy-5.2/astropy/config/configuration.py	2022-12-12 11:57:50.000000000 +0100
+++ astropy-5.2.config/astropy/config/configuration.py	2023-01-07 00:39:52.705836236 +0100
@@ -20,7 +20,7 @@
 from textwrap import TextWrapper
 from warnings import warn
 
-from astropy.extern.configobj import configobj, validate
+import configobj, validate
 from astropy.utils import find_current_module, silence
 from astropy.utils.decorators import deprecated
 from astropy.utils.exceptions import AstropyDeprecationWarning, AstropyWarning
