diff -ur astropy-0.3rc1/astropy/config/configuration.py astropy-0.3rc1.configobj/astropy/config/configuration.py
--- astropy-0.3rc1/astropy/config/configuration.py	2013-11-19 12:36:12.001071643 +0100
+++ astropy-0.3rc1.configobj/astropy/config/configuration.py	2013-11-19 12:37:28.962673971 +0100
@@ -24,7 +24,7 @@
 from warnings import warn
 
 from ..extern import six
-from ..extern.configobj import configobj, validate
+import configobj, validate
 from ..utils.exceptions import AstropyWarning
 
 
