diff -ur astropy-4.3.1/astropy/config/configuration.py astropy-4.3.1.config/astropy/config/configuration.py
--- astropy-4.3.1/astropy/config/configuration.py	2021-08-11 22:17:12.000000000 +0200
+++ astropy-4.3.1.config/astropy/config/configuration.py	2021-09-09 20:46:52.753940849 +0200
@@ -20,7 +20,7 @@
 from warnings import warn
 from contextlib import contextmanager
 
-from astropy.extern.configobj import configobj, validate
+import configobj, validate
 from astropy.utils import find_current_module, silence
 from astropy.utils.exceptions import AstropyDeprecationWarning, AstropyWarning
 from astropy.utils.compat.context import nullcontext
