diff -ur astropy-0.4.2/astropy/config/tests/test_configs.py astropy-0.4.2.conf/astropy/config/tests/test_configs.py
--- astropy-0.4.2/astropy/config/tests/test_configs.py	2014-12-05 00:03:10.533260470 +0100
+++ astropy-0.4.2.conf/astropy/config/tests/test_configs.py	2014-12-05 00:05:13.581269048 +0100
@@ -107,7 +107,7 @@
     while apycfg.parent is not apycfg:
         apycfg = apycfg.parent
     f = tmpdir.join('astropy.cfg')
-    with io.open(f.strpath, 'w', encoding='utf-8') as fd:
+    with io.open(f.strpath, 'wb') as fd:
         apycfg.write(fd)
     with io.open(f.strpath, 'rU', encoding='utf-8') as fd:
         lns = [x.strip() for x in f.readlines()]
