diff -ur astropy-4.3.1/astropy/coordinates/angle_formats.py astropy-4.3.1.ply/astropy/coordinates/angle_formats.py
--- astropy-4.3.1/astropy/coordinates/angle_formats.py	2021-08-11 22:17:12.000000000 +0200
+++ astropy-4.3.1.ply/astropy/coordinates/angle_formats.py	2021-09-09 21:08:11.871724229 +0200
@@ -75,7 +75,7 @@
 
     @classmethod
     def _make_parser(cls):
-        from astropy.extern.ply import lex, yacc
+        from ply import lex, yacc
 
         # List of token names.
         tokens = (
diff -ur astropy-4.3.1/astropy/utils/parsing.py astropy-4.3.1.ply/astropy/utils/parsing.py
--- astropy-4.3.1/astropy/utils/parsing.py	2021-08-11 22:17:12.000000000 +0200
+++ astropy-4.3.1.ply/astropy/utils/parsing.py	2021-09-09 21:08:42.559223258 +0200
@@ -84,7 +84,7 @@
     reflags : int
         Passed to ``ply.lex``.
     """
-    from astropy.extern.ply import lex
+    from ply import lex
 
     caller_file = lex.get_caller_module_dict(2)['__file__']
     lextab_filename = os.path.join(os.path.dirname(caller_file), lextab + '.py')
@@ -137,7 +137,7 @@
         the output file. This is inserted into a comment in the generated
         file.
     """
-    from astropy.extern.ply import yacc
+    from ply import yacc
 
     caller_file = yacc.get_caller_module_dict(2)['__file__']
     tab_filename = os.path.join(os.path.dirname(caller_file), tabmodule + '.py')
