diff -ur astropy-0.3rc1/astropy/tests/helper.py astropy-0.3rc1.pytest/astropy/tests/helper.py
--- astropy-0.3rc1/astropy/tests/helper.py	2013-11-19 12:58:43.970955591 +0100
+++ astropy-0.3rc1.pytest/astropy/tests/helper.py	2013-11-19 13:00:09.510620451 +0100
@@ -34,7 +34,7 @@
 
 from .. import test
 
-if os.environ.get('ASTROPY_USE_SYSTEM_PYTEST') or '_pytest' in sys.modules:
+if True: # Use system pytest always, modified in the Fedora package
     import pytest
 
 else:
