diff -ur astropy-0.3.2/astropy/wcs/setup_package.py astropy-0.3.2.wcslib/astropy/wcs/setup_package.py
--- astropy-0.3.2/astropy/wcs/setup_package.py	2014-05-21 11:19:24.260042437 +0200
+++ astropy-0.3.2.wcslib/astropy/wcs/setup_package.py	2014-05-21 11:20:53.103770423 +0200
@@ -309,10 +309,11 @@
         'wcsmath.h',
         'wcsprintf.h',
         ]
-    for header in wcslib_headers:
-        shutil.copy(join('cextern', 'wcslib', 'C', header),
+    if not setup_helpers.use_system_library('wcslib'):
+        for header in wcslib_headers:
+            shutil.copy(join('cextern', 'wcslib', 'C', header),
                     join('astropy', 'wcs', 'include', 'wcslib', header))
-        api_files.append(join('include', 'wcslib', header))
+            api_files.append(join('include', 'wcslib', header))
 
     return {
         str('astropy.wcs.tests'): ['data/*.hdr', 'data/*.fits',
